kv/dist_sender_test.go

```diff
@@ -209,11 +212,17 @@ func TestSendRPCOrder(t *testing.T) {
 			Replicas: nil,
 		}
 		for i := int32(1); i <= 5; i++ {
-			addr := util.MakeRawAddr("tcp", fmt.Sprintf("%d", i))
+			addr, err := net.ResolveTCPAddr("tcp", fmt.Sprintf("localhost:%d", i))
```

Comment:
I'd prefer using bogus names to make it clear nobody is actually connecting there. Something like `fmt.Sprintf("node%d", i)`.